deepchem/models/torch_models/IRV.py

```diff
@@ -0,0 +1,197 @@
+from deepchem.models.torch_models.torch_model import TorchModel
+from deepchem.models.losses import SigmoidCrossEntropy
+import torch
+import torch.nn as nn
+from typing import Callable, Optional
+
+if torch.cuda.is_available():
```

Comment:
Please remove this code; this is dangerous and has a global effect!

Reply:
removed